web/src/features/charts/bar-breakdown/utils.ts

```diff
@@ -194,17 +193,3 @@ export const getExchangesToDisplay = (
     (exchangeZoneKey) => !exchangeZoneKeysToRemove.has(exchangeZoneKey)
   );
 };
-
-export const useHeaderHeight = () => {
-  const [headerHeight, setHeaderHeight] = useState<number>(0);
-
-  useEffect(() => {
-    const headerElement = document.querySelector('header');
-    if (headerElement) {
-      const height = headerElement.offsetHeight;
-      setHeaderHeight(height * 1.1);
-    }
-  }, [window.innerWidth, window.innerHeight]);
-
-  return headerHeight;
-};
```

Comment:
Can you remove the useEffect and useState imports in this file that are no longer needed?